sass/base/_helper-classes.scss

```diff
@@ -70,6 +70,9 @@
 .realtime {
   color: $realtime-color;
 }
+.late {
+  color: $late-color!important;
```

Comment:
Is it necessary to use `!important`? IMO, this should be avoided at all costs: https://stackoverflow.com/questions/9245353/what-does-important-mean-in-css